docs/core/base/document.md

```diff
@@ -0,0 +1,112 @@
+# Document
+
+The term "Document" is used in the only in the code. It refers a single piece of content that created by Elementor, such as a post, page, or a saved template.
+
+Each document has its own type and features.
+
+A document is saved as a `post` in the WordPress `wp_posts` table. His elements and settings are saved as metadata in the `wp_postmeta` table.
+
+In PHP, the document is represented by the `Elementor\Base\Document` class.
+
+This class is like a DB Model that knows how to save and load its data.
+
+### Why "Document"?
+
+WordPress' content are called `posts` it's an historical term that came from the days of WordPress as a blogging system.
+
+In Elementor, it can be many document types for one `post_type` like the saved templates Page and Section that are both a `elementor_library` post type.
+
+The term `document` is used instead, as the browser called his DOM content as a "document".
```

Comment:
It doesn't really explain why "document" is a  better name than "post", and I don't really understand how it's related to the browser if it's a DB entity

Reply:
In order to not be confused with WP post.